scripts/invalidate-just-cache.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const topLevelDep = path.join(__dirname, '../packages/prettier-rules/node_modules/.just');
```

Comment:
Why does this pull from `prettier-rules` ?

Reply:
This file will go away in my next PR - but it just so happens to be the one package that is at the top level (so invalidating this cache invalidates all build caches for all packages)